twpca/model.py

```diff
@@ -415,6 +425,24 @@ def predict(self, X=None):
 
             return pred
 
+    def dump(self):
+        """Serializes model variables"""
+        return self._sess.run(self._vars)
+
+    def load(self, new_vars):
+        """Assigns model variables from numpy arrays"""
+        self._sess.run([tf.assign(self._vars[k], v) for k, v in new_vars.items()])
```

Comment:
You load from a dictionary (`new_vars` is a dictionary like object here), but dump to a list?